src/rng.h

```diff
@@ -10,4 +12,12 @@ int dice(int number, int sides);
 
 int djb2_hash(const unsigned char *input);
 
+inline int djb2_hash(char const *input) {
```

Comment:
Standard protocol for DDA: if it's not being used for something, it shouldn't be in.  I'm sure we've missed something somewhere, but we don't add new stuff without a use.